scripts/countEnglishWordsSince.ts

```diff
@@ -42,21 +50,14 @@ rl.question('Which revision sould we compare: ', commitId => {
 const findModifiedStrings = (prev: StringRecord, curr: StringRecord): string[] => {
   const modifiedStrings: string[] = []
   for (const key in curr) {
-    const currentValue = curr[key]
-    const previousValue = prev[key]
+    const currentValue = curr?.[key]
+    const previousValue = prev?.[key]
```

Comment:
Nice catch! This is a sign our type system is failing us, might be a good idea to add undefined as a possible value type in `StringRecord` and work through any other type error it picks up

Reply:
Is it? I've actually started with this idea but `undefined` is only a "possible" value during the comparison because a key can be missing in both direction (new strings, deleted strings)... so I figured it wasn't really part of that type/interface.